### PR TITLE
Soarian R4 FHIR OperationOutcome Documentation 

### PR DESCRIPTION
**Pull request by DeepakGupta93**
Description
----
Review Request for Soarian R4 FHIR OperationOutcome Documentation

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
OO_1
OO_2
OO_3
OO_4
OO_5

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/soarian/r4/foundation/other/operationoutcome.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
+---
+title: Operation Outcome | R4 API
+---
```
Comment (SachinPai04) [inline content/soarian/r4/foundation/other/operationoutcome.md:3]: Remove space from "Operation Outcome" .

Comment (DeepakGupta93) [reply content/soarian/r4/foundation/other/operationoutcome.md:3]: Removed Space

File: content/soarian/r4/foundation/other/operationoutcome.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+OperationOutcome.issue.__code__	          | not-found
+OperationOutcome.issue.__details__	      | No records are associated with the ID : [%s] due to a patient merge action. Target patient ID: [%s].
+
+When too many search results qualify for a given search criteria:
+
+Element                                   | Value                                    
+------------------------------------------|------------------------------------------------------------------------------------------------------
+OperationOutcome.issue.__severity__	      | error
+OperationOutcome.issue.__code__	          | business-rule
+OperationOutcome.issue.__details__	      | More than 499 search results qualify for [%s] resource. Please narrow the search.
```
Comment (HRNWriter) [inline content/soarian/r4/foundation/other/operationoutcome.md:105]: Change "please narrow the search." to "Narrow the search."

PRODOC-160857.

File: content/soarian/r4/foundation/other/operationoutcome.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+When a persona is missing in the JWT validation claims:
+
+Element                                   | Value                                    
+------------------------------------------|------------------------------------------------------------------------------------------------------
+OperationOutcome.issue.__severity__	      | error
+OperationOutcome.issue.__code__	          | security
+OperationOutcome.issue.__details__	      | Incorrect persona.
+
+When a resource is requested for which the user does not have access:
```
Comment (HRNWriter) [inline content/soarian/r4/foundation/other/operationoutcome.md:165]: Change "for which" to "to which".

PRODOC-160857.

**Review by khansasultana — APPROVED** [review]

**Review by slvsridhar — APPROVED** [review]

**Review by vf056083 — APPROVED** [review]

**Review by VamsiMure — APPROVED** [review]

**Pull request merged**